### PR TITLE
add teal.gallery ref to readme

### PR DESCRIPTION
**Pull request by pawelru**

### CODE REVIEW

**Comment by nikolas-burkoff** [comment]
Should we also add to tmg and tmc?

File: README.md [hunk 1]
```diff
@@ -50,6 +50,8 @@ A lot of the functionality of the `teal` framework derives from the following pa
 
 See these packages for more information about how to use the different parts of the `teal` framework.
 
+Please see [`teal` gallery](https://github.com/insightsengineering/teal.gallery) and [TLG Catalog](https://github.com/insightsengineering/tlg-catalog) to see examples of `teal` apps.
```
Comment (nikolas-burkoff) [inline README.md:53]: Is https://insightsengineering.github.io/teal.gallery/main/ better?

**Comment by pawelru** [comment]
> Should we also add to tmg and tmc, t.o and t.g?

tmc, tern are done
tmg pr opened

File: README.md (hunk 1 above)
Comment (pawelru) [reply README.md:53]: hmmm you might be right but currently all our references are to the GH (not GitHubPages) so I wanted to stay consistent to the currently existing approach

**Review by nikolas-burkoff — APPROVED** [review]

**Pull request merged**